hadoop-ozone/ozonefs-shaded/pom.xml

```diff
@@ -187,6 +304,87 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>${maven-dependency-plugin.version}</version>
```

Comment:
nit: version can be omitted, already defined in `pluginManagement` in root `pom.xml`.

Reply:
Done.